src/main/java/org/ambraproject/rhino/config/HibernateLoggingInterceptor.java

```diff
@@ -88,19 +89,13 @@ public boolean onSave(Object entity, Serializable id, Object[] state, String[] p
     if (entity instanceof Comment) {
       Comment comment = (Comment) entity;
       sendMessage("ambra-comment", comment.getCommentUri(), KafkaEventType.ADD);
+    } else if (entity instanceof ArticleRevision) {
+      ArticleRevision revision = (ArticleRevision) entity;
+      sendMessage("ambra-article", revision.getIngestion().getArticle().getDoi(), KafkaEventType.ADD);
     }
     return super.onSave(entity, id, state, propertyNames, types);
   }
 
-  @Override
-  public void onDelete(Object entity, Serializable id, Object[] state, String[] propertyNames, Type[] types) {
```

Comment:
We actually never 'delete' comments - we only ever set the `isRemoved` flag in the DB, which simply replaces the comment text with 'removed'.

We have not talked about keeping comments up to date like this after PATCH or UPDATE events, so I will leave that implementation to another ticket.